vent/sqldb/adapters/adapter.go

```diff
@@ -0,0 +1,15 @@
+package adapters
+
+import "github.com/monax/bosmarmot/vent/types"
+
+// DBAdapter database acces interfase
+type DBAdapter interface {
```

Comment:
mispelling 
`// DBAdapter database acces interfase` should be
`// DBAdapter database access interface`